docs/concepts/trading-on-vega/orders.md

```diff
@@ -106,11 +107,39 @@ Unparked pegged orders will be rejected:
 * If the reference price no longer exists (e.g. no best bid)
 * If the price moves to a value that means it would create an invalid order if the offset was applied
 
+### Iceberg order
+An iceberg order is a limit order for a large amount that, rather than being entered as a single large order of that size, is placed on the book as a smaller order that is replenished as that order amount is filled. The peak / 'visible' amount can be filled with one trade, while the reserve is used to support the smaller order amount.
+
+In other words, an iceberg order allows for the smaller order amount, known as the peak size, to be replenished from a 'hidden' total size. An iceberg's peak size is replenished in full until the whole iceberg's volume trades away, or the order is cancelled or expires. It's replenished whenever the peak drops below the minimum visible size, set when submitting the order. If the reserve amount becomes lower than the peak size, the peak size is then set to the full remaining amount, with nothing in reserve.
```

Comment:
If we're saying the "peak_size" is the static value which is the most the "visible size" can be, should here:
"It's replenished whenever the peak drops below the minimum visible size"
it instead be:
"It's replenished whenever the remaining visible size drops below the minimum visible size"?

Reply:
Maybe also here:
"An iceberg's peak size is replenished in full"

should instead be:
"An iceberg's visible size is replenished in full to the peak size"